index.html

```diff
@@ -38,5 +38,95 @@ <h1> <span class="wrapper">I'm Mariane</span> <span class="wrapper">Glad to see<
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="card-4">
+            <div class="card-4-bg"></div>
+            <h2>Tonic</h2>
+            <div class="canopy"></div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="lang-btn">
+               <ul>
+                  <li><a href="#"><button>html</button></a></li>
+                  <li><a href="#"><button>css</button></a></li>
+                  <li><a href="#"><button>javaScript</button></a></li>
+               </ul>
+            </div>
+            <div class="card-1-btn"><button>See Project</button></div>
+        </section>
+        <section class="ams">
+            <h1>About Myself</h1>
+            <p>
+               Hello I'm a software developer! I can help you build a product, feature or website
+               Look through some of my work and experiance! If you like what you see and have a project you need coded,
+               don't hestiate to contact me.
+            </p>
+            <div class="letconnect_text">LET'S CONNECT</div>
+            <div class="letconnect_icons">
+               <ul>
+                  <li><a href="#"><img src="Images\Vector4.svg" alt="" width="17.99px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector2.svg" alt="" width="20px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector5.svg" alt="" width="14.17px" height="20px"></a></li>
+                  <li><a href="#"><img src="Images\Vector1.svg" alt="" width="19.99px" height="19.96px"></a></li>
+                  <li><a href="#"><img src="Images\Vector3.svg" alt="" width="20px" height="16px"></a></li>
+               </ul>
+            </div>
+            <button>Get my resume</button>
+            <div>
+               <ul>
+                  <li><a href="#"><img src="Images\about_languages.svg" alt="" width="327px" height="72px"></a></li>
+                  <ul>
+                     <li><a href="#"><img src="Images\about_javascript.svg" alt="" width="303px"
+                              height="66.72px"></a></li>
+                     <li><a href="#"><img src="Images\about_html.svg" alt="" width="303px" height="66.72px"></a></li>
+                     <li><a href="#"><img src="Images\about_css.svg" alt="" width="303px" height="66.72px"></a></li>
+                  </ul>
+                  <li><a href="#"><img src="Images\about_frameworks.svg" alt="" width="327px" height="72px"></a></li>
+                  <li><a href="#"><img src="Images\about_skills.svg" alt="" width="327px" height="72px"></a></li>
```

Comment:
- [ ] According to the figma template, kindly make sure all the above code are images as it should be `text` and images. Images can be only `logos` and `chevrons icons` but others should be typed and styled correctly as right now try to increase the size of your screen but on mobile width your design dosn't match the one from the template.
![Screenshot from 2023-03-24 19-39-19](https://user-images.githubusercontent.com/83042742/227662988-0bed3ed9-05f3-4977-accc-395f83b8694f.png)